.env.dev

```diff
@@ -4,10 +4,12 @@ VERSION=dev
 DATABASE_USER=docker
 DATABASE_PASSWORD=docker
 DATABASE_DB=airbyte
+DEV_ROOT_PARENT=/tmp
 DEV_ROOT=/tmp/dev_root
 CONFIG_ROOT=/tmp/data
 WORKSPACE_ROOT=/tmp/workspace
-WORKSPACE_DOCKER_MOUNT=dev-workspace
+WORKSPACE_DOCKER_MOUNT=/tmp/dev_root/workspace
```

Comment:
when we are mounting directories `*_MOUNT` needs to be the same as `*ROOT`. e.g. `LOCAL_DOCKER_MOUNT=/tmp/airbyte_local` and `LOCAL_ROOT=/tmp/airbyte_local` need to be the same directory.